src/Resources/public/js/configuration/graphql/configItem.js

```diff
@@ -15,11 +15,16 @@ pimcore.registerNS("pimcore.plugin.datahub.configuration.graphql.configItem");
 pimcore.plugin.datahub.configuration.graphql.configItem = Class.create(pimcore.element.abstract, {
 
     saveUrl: "/admin/pimcoredatahub/config/save",
+    disableForm: false,
 
     initialize: function (data, parent) {
         this.parent = parent;
         this.data = data.configuration;
         this.modificationDate = data.modificationDate;
+        if(data['configuration']['general']['writeable'] != null &&
+            data['configuration']['general']['writeable'] != undefined) {
+            this.disableForm = !data['configuration']['general']['writeable'];
```

Comment:
writeable check should be negated here..right?

```suggestion
            this.disableForm = data['configuration']['general']['writeable'];
```

Reply:
Nevermind...it is used in the disabled property so makes sense!

Reply:
Ok...save button should consider this check at line 111

Reply:
When we disable the save button, the user would never get feedback on why he can´t save the configuration. Should we really do this? Wdyt?

Reply:
We've done this for thumbnails configuration:
![image](https://user-images.githubusercontent.com/5137917/140729648-e8071c1d-2991-4ca8-b4ae-66ec815bbaca.png)

And it is better than making every field read only and then consistent with other features.